docs/dev/README.md

```diff
@@ -0,0 +1,70 @@
+# Quick start guide for Assisted Service Developer
+
+Over time, assisted-service has been adapted to different scenarios, and has become one more piece of many other subsystems in which it works integrated with other Openshift components.
+
+For this reason, depending on the development we are working on, we have different scenarios for developers that allow us greater flexibility for our daily work. For example, there will be cases in which we are only interested in doing some specific tests on the API and this can be covered with a light environment, but in other cases we may need a complete Openshift cluster with its control plane and worker nodes, in which case the scenario will be more complex and heavy.
+
+The frequently used scenarios are described below.
+
+## Testing simple changes to the REST service
+[Detailed documentation](../../deploy/podman/README.md)
+
+In this scenario the goal is to run small tests with the assisted-installer REST API. This can be done in a local environment by starting the assisted-service with podman.
+
+We can start it directly with make:
+```shell
+make deploy-onprem
+```
+
+## Testing e2e-flow with REST
+[Detailed documentation](https://github.com/openshift/assisted-test-infra/blob/master/README.md)
+
+This scenario is useful for developing and testing our end-to-end tests.
+This environment requires a large amount of resources since it will deploy additional baremetal hosts (VMs), so **running it in local environments is not recommended**.
+
+We can start this scenario from the `assisted-test-infra` project:
+```shell
+make run
+```
+
+## Testing kube-api features
+[Detailed documentation](http://crc.dev/crc/getting_started/getting_started/introducing/)
+
+The goal here would be to test the operation of the resources associated with assisted-installer through the kube-api. This is a lightweight scenario and we could work with a local environment.
+
+We can install a local environment (Openshift / OKD / Microshift) with the **crc** tool. There is a section to perform this type of installation in the [Openshift console](https://console.redhat.com/openshift/create/local).
```

Comment:
What's the difference between this scenario and the one below, `Testing infrastructure operator`?
Can't it be unified under the same point?